doc/api/magic-plot.md

```diff
@@ -160,3 +160,100 @@ ax = %sqlplot histogram --table no_nulls --column body_mass_g --with no_nulls
 ax.set_title("Body mass (grams)")
 _ = ax.grid()
 ```
+## `%sqlplot bar`
+
+Shortcut: `%sqlplot bar`
+
+`-t`/`--table` Table to use (if using DuckDB: path to the file to query)
+
+`-c`/`--column` Column to plot.
+
+`-o`/`--orient` Barplot orientation (`h` for horizontal, `v` for vertical)
+
+`-w`/`--with` Use a previously saved query as input data
+
+`-sn`/`--show_num` Show number on top of the bar
+
+Barplot does not support NULL values, so let's remove them:
```

Comment:
can we add support for NULL values? you can run a query to filter them first

Reply:
Should I raise some warning when there is NULL?

Reply:
good point. not a warnings.warn, but printing a message saying that we're removing NULLs makes sense

Reply:
Should the message be printed only when there is NULL or a generic message?
Currently added a general message:

```python
print(f"Removing NULLs, if there exists any from {column}")
```

Reply:
Implemented `2` option + Generic null Message for info :

Sample SQL query:
```SQL
select "{{x_}}" as x,
"{{height_}}" as height
from "{{table}}"
where "{{x_}}" is not null
and "{{height_}}" is not null;
```